input/pagecontent/terminology.md

```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
-- [Medicine Substitution Reason](https://healthterminologies.gov.au/fhir/ValueSet/medicine-substitution-reason-1)
-- [Participation in Health Program](https://healthterminologies.gov.au/fhir/ValueSet/participation-in-health-program-1)
-- [Pathology Diagnostic Service Category](https://healthterminologies.gov.au/fhir/ValueSet/pathology-diagnostic-service-category-1)
-- [Procedure](https://healthterminologies.gov.au/fhir/ValueSet/procedure-1)
-- [RCPA SPIA Pathology Reporting](https://healthterminologies.gov.au/fhir/ValueSet/spia-pathology-reporting-1)
-- [RCPA - SPIA Chemical Pathology Terminology Reference Set](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/spia-chemical-pathology-refset-3?ui:source=search)
-- [RCPA - SPIA Haematology Terminology Reference Set](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/spia-haematology-refset-3?ui:source=search)
-- [RCPA - SPIA Immunopathology Terminology Reference Set](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/spia-immunopathology-refset-3?ui:source=search)
-- [RCPA - SPIA Microbiology Serology Molecular Pathology Terminology Reference Set](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/spia-microbiology-serology-molecular-refset-3?ui:source=search)
-- [Reason For Non Participation In Health Program](https://healthterminologies.gov.au/fhir/ValueSet/reason-non-participation-health-program-1)
-- [Reason for Request](https://healthterminologies.gov.au/fhir/ValueSet/reason-for-request-1)
-- [Reason Vaccine Administered](https://healthterminologies.gov.au/fhir/ValueSet/reason-vaccine-administered-1)
-- [Reason Vaccine Not Administered](https://healthterminologies.gov.au/fhir/ValueSet/reason-vaccine-not-administered-3)
-- [Related Person Relationship Type](https://healthterminologies.gov.au/fhir/ValueSet/related-person-relationship-type-1)
-- [Route of Administration](https://healthterminologies.gov.au/fhir/ValueSet/route-of-administration-1)
-- [Separation Mode](https://healthterminologies.gov.au/fhir/ValueSet/separation-mode-1)
-- [Service Type](https://healthterminologies.gov.au/fhir/ValueSet/service-type-1)
-- [Specimen Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-type-1)
-- [Specimen Collection Procedure](https://healthterminologies.gov.au/fhir/ValueSet/specimen-collection-procedure-1)
-- [Specimen Container Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-container-type-1)
-- [Substance](https://healthterminologies.gov.au/fhir/ValueSet/substance-1)
-- [Vaccination Target Disease](https://healthterminologies.gov.au/fhir/ValueSet/vaccination-target-disease-1)
+|ValueSet|Profile where used|Published|
+|---|---|---|
+|[Accession Number Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-accession-number-type.html)|[AU Accession Number](StructureDefinition-au-accessionnumber.html)|AU Base|
```

Comment:
For any AU Base value set use the relative URL not absolute, e.g. 
instead of: https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-accession-number-type.html
use: ValueSet-accession-number-type.html

Reply:
Amended all instances, thanks.